public/app/features/explore/Graph.test.tsx

```diff
@@ -8,6 +8,7 @@ const setup = (propOverrides?: object) => {
     size: { width: 10, height: 20 },
     data: mockData().slice(0, 19),
     range: { from: 'now-6h', to: 'now' },
+    timeZone: { raw: 'browser', isUtc: false },
```

Comment:
Would be okay to add a constant `DefaultTimeZone` to grafana/ui?

Reply:
So for example a constant and a factory method in grafana/ui/types:

```typescript
export const parseTimeZone = (raw: string) => {
  return {
    raw: raw,
    isUtc: raw === 'utc',
  } as TimeZone;
};

export const DefaultTimeZone = parseTimeZone('browser');
```

Reply:
@torkelo @hugohaggmark any thoughts?

Reply:
```ts
export const parseTimeZone = (raw: string): TimeZone => {
  return {
    raw: raw,
    isUtc: raw === 'utc',
  };
};

export const DefaultTimeZone = parseTimeZone('browser');
```
Maybe move TimeZone to function declaration and skip the `as TimeZone part`.

Reply:
Nice!